lib/tasks/pacbio_data.rake

```diff
@@ -162,6 +162,32 @@ namespace :pacbio_data do
     end
     print COMPLETED
 
+    print "   -> Creating runs for #{v12_sequel_iie.name}..."
+    pool_records.zip(pools).each_with_index do |(pool, tag_set), i|
+      Pacbio::Run.create!(
+        name: "RUN-#{v12_sequel_iie.name}-#{tag_set[:tag_set]}x#{tag_set[:size]}",
+        system_name: Pacbio::Run.system_names['Sequel IIe'],
+        smrt_link_version: v12_sequel_iie,
+        plates: [Pacbio::Plate.new(
+          sequencing_kit_box_barcode: '130429101826100021624',
+          plate_number: 1,
+          wells: [Pacbio::Well.new(
+            pools: [pool],
+            row: 'A',
+            column: i + 1,
+            ccs_analysis_output_include_kinetics_information:	'No',
+            ccs_analysis_output_include_low_quality_reads:	'No',
+            include_fivemc_calls_in_cpg_motifs:	'Yes',
+            demultiplex_barcodes:	'In SMRT Link',
+            on_plate_loading_concentration: 1,
+            binding_kit_box_barcode: '030425102194100010424',
+            movie_time: '20.0'
+          )]
+        )]
+      )
+    end
+    print COMPLETED
```

Comment:
```suggestion
    puts COMPLETED
```
Maybe? So line 191 doesn't end up on the same line (I think that's how it works!?)